### PR TITLE
enable requestServingIsolation test on hypershift periodics 4.16-4.18

### PR DESCRIPTION
**Pull request by Patryk-Stefanski**

### CODE REVIEW

File: ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.15__periodics.yaml [hunk 1]
```diff
@@ -45,6 +45,9 @@ tests:
   cron: 0 */6 * * *
   steps:
     cluster_profile: hypershift
+    env:
+      ENABLE_HYPERSHIFT_CERT_ROTATION_SCALE: "true"
```
Comment (sjenning) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.15__periodics.yaml:49]: not this. just ` REQUEST_SERVING_COMPONENT_TEST`.

**Comment by sjenning** [comment]
/approve
/lgtm
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-aws-ovn

**Comment by sjenning** [comment]
/hold
until https://github.com/openshift/hypershift/pull/4842 merges

**Comment by Patryk-Stefanski** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.18-periodics-e2e-aws-ovn

**Comment by Patryk-Stefanski** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.17-periodics-e2e-aws-ovn

**Comment by Patryk-Stefanski** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.16-periodics-e2e-aws-ovn

**Pull request merged**